nin/frontend/app/scripts/directives/keybinding.js

```diff
@@ -32,7 +32,9 @@ angular.module('nin').directive('keybinding', function(commands, render, demo) {
     },
     '13': function(e) {
       // 'return'
-      commands.jumpToFrame(0);
+      var cuePoint = {};
+      commands.getCuePoint(cuePoint);
```

Comment:
How about using a callback function instead?
It's a common pattern and then we won't have to use the proxy object!